nipoppy/extractors/freesurfer/run_FS_utils.py

```diff
@@ -16,10 +16,13 @@ def get_mris_preproc_cmd(FS_dir, participants_list, out_file, meas="thickness",
     """
     participants_str_list = []
     for participant in participants_list:
-        dirpath = Path(f"{FS_dir}/{participant}")
-        dirpath_status = Path.is_dir(dirpath)
-        if dirpath_status:
+        fpath_lh = Path(f"{FS_dir}/{participant}/surf/lh.thickness")
+        fpath_rh = Path(f"{FS_dir}/{participant}/surf/rh.thickness")
+        fpath_status = Path.is_file(fpath_lh) & Path.is_file(fpath_rh)
+        if fpath_status:
             participants_str_list.append(f"--s {participant}")
+        else:
+            print(f"ignoring {participant} with missing surf files...")
```

Comment:
Is there a reason why we are not using the Freesurfer bagel to get the list of participants who have full output files?

Reply:
I guess we could check against it. But I think it's better to keep extractors as stand-alone as possible since they might be used on a subset (e.g. Dx / sex etc) in a dataset or also on pooled subjects across datasets which makes things complicated.